R/fct_cross.R

```diff
@@ -0,0 +1,45 @@
+#' Combine levels from two or more factors to create new factor
+#'
+#' Computes a factor whose levels are all the combinations of the levels of the input factors.
+#'
+#' @param .f A factor or character vector
+#' @param ...  additional factors or character vectors
+#' @param sep A character string to separate the levels
+#' @param keep_empty If TRUE, keep combinations with no observations as levels
+#' @return The new factor
+#'
+#' @export
+#' @examples
+#' fruit <- factor(c("apple","kiwi","apple","apple"))
+#' colour <- factor(c("green","green","red","green"))
+#' eaten <- c("yes","no","yes","no")
+#' fct_cross(fruit, colour)
+#' fct_cross(fruit, colour, eaten)
+#' fct_cross(fruit, colour, keep_empty=TRUE)
+
```

Comment:
Can you please use the tidyverse style? Easiest way is probably to use styler package

Reply:
Cool. I didn't know about the styler add-in.